### PR TITLE
Fixes the alignment of "Create Account" on the login screen 

### LINKED ISSUE #271: Alignment issue- while minimizing browser
Test environment : 
Affected IS version: wso2is-5.10.0-alpha2-SNAPSHOT
Browser: Google chrome

image

### PR DESCRIPTION
**Pull request by thivi**
image

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/basicauth.jsp [hunk 1]
```diff
@@ -314,6 +314,7 @@
                     <%=AuthenticationEndpointUtil.i18n(resourceBundle, "continue")%>
             </button>
         </div>
+        <div></div>
```
Comment (DimalChandrasiri) [inline apps/authentication-portal/src/main/webapp/basicauth.jsp]: Shall we format the divs to increase readability? WDYT?

Comment (jeradrutnam) [reply apps/authentication-portal/src/main/webapp/basicauth.jsp]: This an unwanted line. Better remove it.

Comment (thivi) [reply apps/authentication-portal/src/main/webapp/basicauth.jsp]: This div tag is needed to clear the float of two elements above. Without this tag, this is how the buttons will be displayed:
image

Comment (jeradrutnam) [reply apps/authentication-portal/src/main/webapp/basicauth.jsp]: You can clear float using css pseudo `:after` property.

**Pull request closed without merging**